ols/app/endpoints/ols.py

```diff
@@ -185,13 +183,16 @@ def conversation_request(llm_request: LLMRequest) -> LLMRequest:
         config.conversation_cache.insert_or_append(
             user_id,
             conversation_id,
-            llm_request.query + "\n\n" + str(llm_response.response or ""),
+            llm_request.query + "\n\n" + str(response or ""),
         )
+    llm_response = LLMResponse(
+        query=llm_request.query, conversation_id=conversation_id, response=response
```

Comment:
to clarify my comment above:  in the finalized api i would expect each of these fields to represent primitive datatypes.  For fields that are strings, i'd expect the content to just be a raw string, not a json representation.

Reply:
let's change it now by dropping the `details` wrapper.  i don't think it will affect much existing code, if any (maybe the gradio ui) and it will avoid confusing people who are building on top of the api and think that we're going to have structured json embedded in these fields.

if it's more complicated than just dropping the details wrapper and putting the response directly into the response field as raw text, then i'm ok not fixing it here and handling it in a follow up, but even as a follow up i'd like to fix it immediately rather than fixing it later as we evolve the api.  I want our basic api to at least reflect the philosophy and intent of our api so everyone is on the same page going forward, and "json blobs in text fields" does not reflect that.

Reply:
I'll do that as a separate PR right after this one.